planner/core/logical_plan_test.go

```diff
@@ -1996,6 +1996,54 @@ func (s *testPlanSuite) TestWindowFunction(c *C) {
 			sql:    "select a from t window w1 as (partition by a) order by (sum(a) over(w1))",
 			result: "TableReader(Table(t))->Window(sum(test.t.a))->Sort->Projection",
 		},
+		{
+			sql:    "select sum(a) over(groups 1 preceding) from t",
+			result: "[planner:1235]This version of MySQL doesn't yet support 'GROUPS'",
```

Comment:
Do we need to keep `MySQL` here? :joy:

Reply:
I don't know, what is your opinion? @zz-jason @morgo

Reply:
How about using "TiDB"?

Reply:
+1 for TiDB in this context.